.github/workflows/build.yml

```diff
@@ -308,3 +308,190 @@ jobs:
           commit-message: publish documentation
           clean-exclude: |
             "version/*"
+
+  upload-nightly:
+    if: (github.ref == 'refs/heads/master') && (github.event_name == 'schedule')
```

Comment:
Did you forget to add the schedule event in "on" section ? For now it is triggering only on push and pull requests.

Reply:
Added